pom.xml

```diff
@@ -23,6 +23,7 @@
         <junit.version>4.12</junit.version>
         <assertj-core.version>3.4.0</assertj-core.version>
         <vertx-core.version>3.2.1</vertx-core.version>
+        <jackson.version>2.7.1</jackson.version>
```

Comment:
Correct. There was a problem with spring. You can update version locally and try it out.